libraries/Microsoft.Bot.Builder.Azure.Blobs/BlobsStorage.cs

```diff
@@ -44,21 +46,39 @@ public class BlobsStorage : IStorage
         /// <para>jsonSerializer.ContractResolver = new DefaultContractResolver().</para>
         /// </param>
         public BlobsStorage(string dataConnectionString, string containerName, JsonSerializer jsonSerializer = null)
+            : this(dataConnectionString, containerName, default, jsonSerializer)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BlobsStorage"/> class.
+        /// </summary>
+        /// <param name="dataConnectionString">Azure Storage connection string.</param>
+        /// <param name="containerName">Name of the Blob container where entities will be stored.</param>
+        /// /// <param name="storageTransferOptions">Used for providing options for parallel transfers <see cref="StorageTransferOptions"/>.</param>
+        /// <param name="jsonSerializer">If passing in a custom JsonSerializer, we recommend the following settings:
+        /// <para>jsonSerializer.TypeNameHandling = TypeNameHandling.None.</para>
+        /// <para>jsonSerializer.NullValueHandling = NullValueHandling.Include.</para>
+        /// <para>jsonSerializer.ContractResolver = new DefaultContractResolver().</para>
+        /// </param>
+        public BlobsStorage(string dataConnectionString, string containerName, StorageTransferOptions storageTransferOptions, JsonSerializer jsonSerializer = null)
```

Comment:
I wonder if it would be better to just add an optional `StorageTransferOptions` parameter to the single constructor.

Reply:
I'm all for it, if we can. I was a little concerned about binary compat...I'm not sure how that works, exactly.